### ISSUE TITLE
Inserted images won't display at equal sizes

### ISSUE THREAD

**Issue by parkerjamesw** [JohnTendik/jtrt-tables#96]
Have a simple table of five columns, one row. In each cell is an image 480x480. No matter what I do, they refuse to display at the same size as each other.

**Comment by JohnTendik** [comment]
That's weird, all the images are the exact same size?

**Comment by parkerjamesw** [comment]
Yeah, no matter what I did, some would display smaller. I finally just took the chart out.
